packages/katello/rubygem-pulp_ansible_client/rubygem-pulp_ansible_client.spec

```diff
@@ -1,73 +1,47 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name pulp_ansible_client
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.13.1
+Name: rubygem-%{gem_name}
+Version: 0.13.2
 Release: 1%{?dist}
-Summary: Pulp 3 Ansible API Ruby Gem
-Group: Development/Languages
+Summary: Pulp 3 API Ruby Gem
 License: GPL-2.0+
-URL: https://github.com/pulp/pulp_ansible/
+URL: https://openapi-generator.tech
```

Comment:
This looks like an upstream issue: https://github.com/pulp/pulp-openapi-generator/pull/71